build/config/phalcon_c_header.php

```diff
@@ -56,6 +56,10 @@
 #include <ext/spl/spl_iterators.h>
 #include <ext/spl/spl_array.h>
 
+#if PHP_VERSION_ID >= 80200
```

Comment:
I don't think the condition is needed.
This is part of the PHP API, always there

The failure on 8.2 may be related to the include of this file in another header.

Reply:
IMHO, better to use:

`#if defined ZEPHIR_USE_PHP_PCRE && ZEPHIR_USE_PHP_PCRE`

Reply:
Agree